next-app/app/schedule/page.js

```diff
@@ -0,0 +1,116 @@
+        }
+    }
+
+    if (scheduleLoading || matchesLoading) {
+        return <Loading />;
+    }
+
+    if (scheduleError || matchesError) {
+        return <Error />;
+    }
+
+    return (
+        <div>
+            <h1 className='display-1 text-center p-auto m-auto pt-2 title'>Schedule</h1>
+            <br />
+            <ScheduleData schedule={scheduleDB} matches={matchesDB} unschedule={unschedule} />
+        </div>
+    );
+}
+
+function ScheduleData({ schedule, matches, unschedule }) {
+    const reformat = reformatMatches(matches);
+
+    return (
+        <div className='container pb-5'>
+            <Accordion >
+                {schedule.map((sval) => {
+                    const unscheduleID = () => { unschedule(sval.id); };
+                    const scheddate = new Date(sval.sdate.slice(0, 10) + 'T00:00:00.000-05:00').toDateString();
+                    return (
+                        <Accordion.Item key={sval.id} eventKey={sval.id - 1}>
+                            <Accordion.Header>{scheddate}</Accordion.Header>
+                            <Accordion.Body>
+                                <table className='table text-center'>
+                                    <thead>
+                                        <tr>
+                                            <th>Match</th>
+                                            <th className='text-primary'>Team A</th>
+                                            <th className='text-primary'>Score</th>
+                                            <th className='text-danger'>Score</th>
+                                            <th className='text-danger'>Team B</th>
+                                        </tr>
+                                    </thead>
+                                    <tbody>
+                                        {reformat.map((mval) => {
+                                            if (mval.mdate === scheddate) {
+                                                return (
+                                                    <tr key={mval.match_id}>
+                                                        <td>Game {mval.match_id}</td>
+                                                        <td>{mval.teamA.join(', ')}</td>
+                                                        <td>{mval.ascore}</td>
+                                                        <td>{mval.bscore}</td>
+                                                        <td>{mval.teamB.join(', ')}</td>
+                                                    </tr>
+                                                );
+                                            }
+                                        })}
+                                    </tbody>
+                                </table>
+                                <div className='text-end'>
+                                    <Button variant='danger' size="lg" onClick={unscheduleID}><i className='bi bi-trash-fill'></i></Button>
```

Comment:
I think we should use a small (or regular) button size here, large takes up a lot of space - almost the same height as a game entry

large (current):
> ![image](https://github.com/user-attachments/assets/f651dcb3-e49a-47b5-9542-32dd38a9d401)

small:
> ![image](https://github.com/user-attachments/assets/e05ae7fe-bbd0-45fe-9a5f-afa986280207)

and maybe a confirmation modal/"are you sure" prompt on the button? I feel like it's warranted given it's a destructive action